core/utils/config/validate.go

```diff
@@ -72,7 +76,9 @@ func validate(v reflect.Value, checkInterface bool) (err error) {
 			// skip the interface if Anonymous, since the parent struct inherits the methods
 			if fe := validate(fv, !ft.Anonymous); fe != nil {
 				if ft.Anonymous {
-					err = multierr.Append(err, fe)
+					for _, e := range utils.UnwrapError(fe) {
+						err = multierr.Append(err, e)
+					}
 				} else {
 					err = multierr.Append(err, NamedMultiErrorList(fe, ft.Name))
```

Comment:
Does this case need to be covered?

Reply:
likely not - I was just testing it to see if that fixed the issue

Reply:
Are you saying that you already tried updating this else block, and it did not fix the issue?